### PR TITLE
Prepare release 1.0.5

### PR DESCRIPTION
**Pull request by shukon**
I think a patch-release would be adequate, since there is one major bugfix and one compatibility fix... @AndreBiedenkapp any objections if I release this as 1.0.5?

### CODE REVIEW

File: pimp/__version__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 """Version information."""
-__version__ = "1.0.4"
+__version__ = "1.0.5_dev"
```
Comment (AndreBiedenkapp) [inline pimp/__version__.py]: For a release we shouldn't mark it as dev any more, right?

Comment (shukon) [reply pimp/__version__.py]: True. I was going to fix that before release, but it probably makes more sense to change in dev-branch, you're right.

**Comment by shukon** [comment]
@AndreBiedenkapp Anything else?

**Comment by AndreBiedenkapp** [comment]
@shukon No everything looks fine now

**Pull request merged**